rust_snuba/src/processors/ourlogs.rs

```diff
@@ -10,7 +10,7 @@ use sentry_arroyo::backends::kafka::types::KafkaPayload;
 use serde::de::{MapAccess, Visitor};
 
 use crate::config::ProcessorConfig;
-use crate::processors::eap_items::{EAPItem, PrimaryKey};
+use crate::processors::eap_items_span::{EAPItemSpan, PrimaryKey};
```

Comment:
is this processor still needed?

Reply:
Yes, we'll still ingest spans this way until we switch Relay to push to the new topic, so we'll have to run both in parallel for a while.